app.py

```diff
@@ -0,0 +1,33 @@
+# Commented out IPython magic to ensure Python compatibility.
+# %cd /spark-data/etl
+
+# %run 'load_data.ipynb'
+# %run 'verify_data.ipynb'
+# %run 'data_cleaning.ipynb'
+# % run 'customer_purchase_behaviour_analysis.ipynb'
+
+from .etl.load_data import load_data,print_schema
+from .etl.verify_data import verify_data_accuracy
+from .etl.data_cleaning import clean_data
+from .utils.read  import show_data
+
+data_files = ["customers", "interactions", "products", "sales_team", "transactions", "country_codes"]
+
+data_dictionary = load_data(data_files)
+
+data_dictionary
+
```

Comment:
are we doing something with this variable ?

Reply:
all the dataframes present in this variable